crates/bin/node/src/logic/inputs.rs

```diff
@@ -37,6 +37,8 @@ pub enum Error {
     Invalid,
     #[error("Proof already used")]
     Used,
+    #[error("amount exceeds max order for keyset {0}: amount {1} exceeds max value {2}")]
```

Comment:
part of this error message is duplicated

Reply:
@tdelabro i'm not quite sure i get/see the duplication you're talking about here

Reply:
"amount exceeds max order for keyset"  and "amount {1} exceeds max value {2}" is twice the same information

I think you can just say "amount {1} exceed max order {2} of keyset {0}"

Reply:
simplify this, rebase and I think it will be good to merge

Reply:
@tdelabro done